packages/workspace/src/browser/workspace-frontend-contribution.spec.ts

```diff
@@ -37,31 +37,24 @@ describe('workspace-frontend-contribution', () => {
 
         ([
 
-            [OS.Type.Linux, 'browser', true, { title, canSelectFiles: true, canSelectFolders: true, filters }],
-            [OS.Type.Linux, 'browser', false, { title, canSelectFiles: false, canSelectFolders: true }],
-            [OS.Type.Linux, 'electron', true, { title, canSelectFiles: true, canSelectFolders: false, filters }],
-            [OS.Type.Linux, 'electron', false, { title, canSelectFiles: false, canSelectFolders: true }],
+            [OS.Type.Linux, 'browser', { title, canSelectFiles: true, canSelectFolders: true, filters }],
+            [OS.Type.Linux, 'electron', { title, canSelectFiles: true, canSelectFolders: false, filters }],
```

Comment:
Question: any reason why some tests were removed ?

Reply:
i will revert this change - as per the short discussion we had in today's dev meeting, the workspace.supportMultiRoot would stay for now. So there is no need to remove tests.